projects/packages/backup/src/js/components/Admin.js

```diff
@@ -205,29 +201,20 @@ const ReviewMessage = connectionLoaded => {
 		return false;
 	};
 
-	// Check if the site has an active Backup subscription older than 3 months
-	const hasThreeMonthsSub = () => {
-		if ( ! Array.isArray( purchases ) || purchases.length === 0 ) {
+	// Check if the last 5 backups were successful
+	const hasFiveSuccessfulBackups = () => {
+		if ( backups.length < 5 || ! Array.isArray( backups ) ) {
```

Comment:
Should this check be reversed so you don't check the length of a non-array?

Reply:
Good point, I just corrected it